src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -173,7 +189,20 @@ class SeekBar extends Slider {
    *         The percentage of media played so far (0 to 1).
    */
   getPercent() {
-    const percent = this.getCurrentTime_() / this.player_.duration();
+    const currentTime = this.getCurrentTime_();
+    let percent;
+    const liveTracker = this.player_.liveTracker;
+
+    if (liveTracker.isLive()) {
+      percent = (currentTime - liveTracker.seekableStart()) / liveTracker.liveWindow();
+
+      // prevent the percent from changing at the live edge
+      if (liveTracker.atLiveEdge()) {
+        percent = 1;
```

Comment:
If the liveTracker  does not report that we are at the live edge the button will not show that we are at the live edge either. This code allows us to be a bit behind the live edge, but still show that we are live to prevent small seek bar movements.